Homework-4/Operations/DictionaryOpeartions.cs

```diff
@@ -0,0 +1,163 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Homework_4.Operations
```

Comment:
We expected that OOPs concepts should have been followed. You should have created an Interface and created 2 separate classes, one for List and other for Dictionary. Both classes should have implemented the interface.